envs/centos7/wheel/Dockerfile-py3.9-10

```diff
@@ -2,17 +2,19 @@ ARG REGISTRY=''
 ARG BASE_TAG=''
 FROM ${REGISTRY}pyprt-base:${BASE_TAG}
 
-# no RPM repos yet, we manually build Python 3.10 with default CentOS 7 toolchain
+ARG PY_VER
+
+# no RPM repos yet, we manually build Python 3.9 and 3.10 with default CentOS 7 toolchain
 WORKDIR /tmp
-ADD https://www.python.org/ftp/python/3.10.6/Python-3.10.6.tar.xz python_src.tar.xz
+ADD https://www.python.org/ftp/python/${PY_VER}.6/Python-${PY_VER}.6.tar.xz python_src.tar.xz

```

Comment:
I would recommend to put the full 3.9 and 3.10 versions into PY_VER. It is coincidence that the micro version `.6` is the same for both 3.9 and 3.10.

Also, we should update to the latest 3.9 which is 3.9.13. The Jenkinsfile cfg.py should be able to transport the full version without additional changes. And there is a corresponding Windows base images [3.9.13-windowsservercore-1809](https://github.com/docker-library/python/blob/6ca54f5e8a180fad4b525c2dfecd23c76ff04894/3.9/windows/windowsservercore-1809/Dockerfile)